docs/crr-pause-resume.md

```diff
@@ -23,17 +23,45 @@ consumed by the Kafka Consumer and are being processed for replication will
 continue to finish replication and will not be paused.
 
 The API will have a Redis instance publishing messages to a specific channel.
-All CRR Kafka Consumers will subscribe to this channel and complete any given
-valid requests.
-
-When a Kafka Consumer pauses, the Consumer is still kept alive and maintains
-any internal state, including offset. The Consumer will no longer be
-subscribed to the CRR topic, so will no longer try consuming any entries.
-When the paused Consumer is resumed, it will again resume consuming entries
-from its last offset.
+QueueProcessors will subscribe to this channel, and on receiving a request
+to pause or resume CRR, will notify all of its BackbeatConsumers to perform the
+action if applicable. If an action occurred, the QueueProcessor will receive an
+update on the current status of each Consumer. Based on the global status of a
+location, the status will be updated in Zookeeper if a change has occurred.
+
+It is important to note, when a Consumer pauses, the Consumer process is still
+kept alive and maintains any internal state, including offset. The Consumer will
+no longer be subscribed to the CRR topic, so will no longer try consuming any
+entries. When the paused Consumer is resumed, it will again resume consuming
+entries from its last offset.
 
 ## Definition of API
 
+* GET `/_/crr/status`
+
+    This GET request checks if cross-region replication is enabled or not for
+    all locations configured as destination replication endpoints.
+
+    Response:
+    ```sh
+    {
+        location1: "disabled",
```

Comment:
Since the output shall be JSON, let's add double-quotes to the keys like `"location1"`